runtime/src/main/java/org/corfudb/runtime/object/transactions/TransactionBuilder.java

```diff
@@ -29,6 +29,25 @@
      */
     public long snapshot = -1L;
 
+
+    /** Require precise conflict detection.
+     *
+     * <p>If set to false, transaction conflict resolution
+     * will rely on the sequencer, which may abort due
+     * to hash collisions.
+     *
+     * <p>If set to true, transaction conflict resolution
+     * will first attempt to resolve using the sequencer
+     * and if the sequencer requests an abort, then we
+     * will retry by reading the log manually. If a true
+     * conflict exists, then we will abort, otherwise
+     * we will reattempt to commit the transaction using
+     * the manually resolved information. Setting
+     * this value to true has significant performance
+     * implications on an abort.
+     */
+    public boolean preciseConflicts = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make preciseConflicts a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)